ThreeBotPackages/zerobot/packages_manager/package.py

```diff
@@ -17,7 +17,7 @@ def start(self):
             website_location.name = "packages"
             website_location.path_url = "/packages"
             website_location.use_jumpscale_weblibs = True
-            fullpath = j.sal.fs.joinPaths(self.package_root, "html/")
```

Comment:
I think we can remove the whole for loop block, as it should be created manually now.

For weblibs, i think the whole `use_jumpscale_weblibs` options can be removed, as it is always available at `/weblibs` (added in webinterface package by default), any package that uses this option need to be updated to use `/weblibs` directly.

Reply:
so `use_jumpscale_weblibs` is no longer needed even in ThreebotPackage location adding?

Reply:
Yes, but if a package expects it to be exposed elsewhere, the package should update its code (e.g. frontend) to use `/weblibs`.